docker-compose.yml

```diff
@@ -1,6 +1,7 @@
-version: '3.9'
+version: '3'
```

Comment:
Was there a reason to demote the docker compose file version?

Reply:
Yes, it wouldnt run on my Vagrant DEV env for some reason.

Reply:
Please investigate it.

Reply:
Investigated.
I needed to explicitly  specify the desired docker-compose version in the Vagrant file.
So according [official docker doc](https://docs.docker.com/compose/release-notes/) , the latest docker-compose version is 1.28.2
I added this value to Vagrant where the provisioning of the docker-compose happens:
compose_version: "1.28.2"

And so we finish up with the following versions on the VM:
```
ubutu@ip-172-31-27-129:~$ docker --version
Dockner version 20.10.3, build 48d30b5
ubuntu@ip-172-31-27-129:~$ docker-compose --version
docker-compose version 1.28.2, build 67630359
```
Which allows us to use the latest compose file format '3.9'. [This doc](https://docs.docker.com/compose/compose-file/compose-versioning/) specifies the compatibility versions